### PR TITLE
[C-5981] Embedded Brand Designer Initial Version

### PR DESCRIPTION
**Pull request by drew-y**
## Description

This PR adds the initial version of the brand designer. This is not meant to be a release version. But provides a foundation to build smaller PRs off of.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: packages/components/src/components/index.tsx [hunk 1]
```diff
@@ -85,20 +86,7 @@ export const CourierComponents: React.FunctionComponent = () => {
         toast: Boolean(toastElement),
       }}
     >
-      {inboxElement &&
-        createPortal(
```
Comment (tk26) [inline packages/components/src/components/index.tsx:89]: do we want to get rid of this?

File: packages/react-brand-designer/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@trycourier/react-brand-designer",
+  "version": "1.17.0",
+  "description": "Emendable brand designer for multi-channel notifications",
```
Comment (tk26) [inline packages/react-brand-designer/package.json:4]:
```suggestion
  "description": "Embeddable brand designer for multi-channel notifications",
```

File: packages/react-brand-designer/package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  "author": "Drew Youngwerth <drew@youngwerth.com>",
```
Comment (tk26) [inline packages/react-brand-designer/package.json:5]: I'd change this to Courier maybe

File: packages/react-brand-designer/package.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  "homepage": "https://github.com/trycourier/courier-react/tree/main/packages/react-elements#readme",
+  "license": "ISC",
```
Comment (tk26) [inline packages/react-brand-designer/package.json:7]: maybe MIT?

File: packages/components/src/components/index.tsx (hunk 1 above)
Comment (drew-y) [reply packages/components/src/components/index.tsx:89]: Yeah. It's fine. That repo is more of a playground than anything.

File: packages/react-brand-designer/package.json (hunk 3 above)
Comment (drew-y) [reply packages/react-brand-designer/package.json]: Good catch. That was a yarn init thing. Removed. (None of the other packages have an author)

File: packages/react-brand-designer/package.json (hunk 4 above)
Comment (drew-y) [reply packages/react-brand-designer/package.json:6]: Inbox uses ISC. I like ISC a little more. Its essentially the same license as MIT but cleaner.

**Review by tk26 — APPROVED** [review]

**Pull request merged**